### PR TITLE
chore(deps) Upgrade eslint

### PR DESCRIPTION
**Pull request by robertpin**
Upgrade eslint related packages
Fix new errors

### CODE REVIEW

File: react/features/base/ui/components/GlobalStyles.web.tsx [hunk 1]
```diff
@@ -12,10 +12,12 @@ import { commonStyles } from '../constants';
...
     const { theme } = useStyles();
 
-    return (<MUIGlobalStyles
-        styles = {{
-            ...commonStyles(theme)
-        }} />);
+    return (
+        <MUIGlobalStyles
+            styles = {{
+                ...commonStyles(theme)
```
Comment (saghul) [inline react/features/base/ui/components/GlobalStyles.web.tsx:18]: can't we remove the extra object here? seems pointless right?

**Review by saghul — DISMISSED** [review]
LGTM with a suggestion

File: react/features/base/ui/components/GlobalStyles.web.tsx (hunk 1 above)
Comment (robertpin) [reply react/features/base/ui/components/GlobalStyles.web.tsx:18]: I think we can

**Review by saghul — APPROVED** [review]

**Comment by robertpin** [comment]
Test failure is unrelated

**Pull request merged**